src/eq_schema/Block.js

```diff
@@ -8,16 +9,63 @@ const pageTypeMappings = {
 };
 
 class Block {
-  constructor(title, description, page) {
+  constructor(title, description, page, ctx) {
     this.id = `block${page.id}`;
     this.title = getInnerHTML(title);
     this.description = getInnerHTML(description);
     this.type = this.convertPageType(page.pageType);
-    this.questions = this.buildQuestions(page);
+    this.questions = this.buildQuestions(page, ctx);
+    if (!isNil(page.routingRuleSet)) {
+      // eslint-disable-next-line camelcase
+      this.routing_rules = this.buildRoutingRules(
+        page.routingRuleSet.routingRules,
+        page.routingRuleSet,
+        page.id,
+        ctx
+      );
+    }
   }
 
-  buildQuestions(page) {
-    return [new Question(page)];
+  buildQuestions(page, ctx) {
+    return [new Question(page, ctx)];
+  }
+
+  buildRoutingRules(routingRules, routingRuleSet, pageId, ctx) {
```

Comment:
I'd be tempted to change this to only pass in the routing rule set, then extract the routingRules and else, which I think are the only things actually being used here.